build/tasks/publish.js

```diff
@@ -94,6 +95,7 @@ gulp.task("npm:ui:sources", function () {
         "src/clarity-angular/main.scss",
         "src/clarity-angular/**/*.clarity.scss"
     ])
+        .pipe(replace('node_modules/bootstrap', '~bootstrap'))
```

Comment:
I think this could be only a temporary solution. 

Ideally, we would want to publish our source code as it is without making changes to it. If users try to build `clarity-ui` by themselves from our published `src`, they would run into the similar problems we've faced.  The problem could be solved automatically once we get off Bootstrap as a dependency.